### PR TITLE
docs: add fruit bar example

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: site/_data/examples.json [hunk 1]
```diff
@@ -512,10 +516,6 @@
         "name": "concat_layer_voyager_result",
         "title": "Comparative Likert Scale Ratings",
         "description": "Comparing Likert scale ratings between two conditions. (Figure 10. from @kanitw et al.'s \"Voyager 2: Augmenting Visual Analysis with Partial View Specifications\" -- http://idl.cs.washington.edu/files/2017-Voyager2-CHI.pdf)."
-      },
-      {
```
Comment (domoritz) [inline site/_data/examples.json]: We had this example already in the gallery.

Comment (kanitw) [reply site/_data/examples.json]: It's ok to show an example twice, as long as they appear in the right sections.

Comment (domoritz) [reply site/_data/examples.json]: It was twice in the same section. 

Screen Shot 2020-03-24 at 15 28 19

Comment (kanitw) [reply site/_data/examples.json]: hahaha ok.

**Pull request merged**